CODEOWNERS

```diff
@@ -3,8 +3,13 @@
 # someone opens a pull request.
 # *           @cube-js/core
 
-/rust/cubesql/                         @cube-js/sql-api
-/rust/cubestore/                       @cube-js/cube-store
+/rust/cubenativeutils                      @cube-js/tesseract
```

Comment:
`/rust/cubenativeutils @cube-js/tesseract` is repeated. Third time is the charm? :smile: